packages/fxa-settings/src/pages/PostVerify/ThirdPartyAuthCallback/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
+import { FluentBundle } from '@fluent/bundle';
+import ThirdPartyAuthCallback from '.';
+import { AppContext } from '../../../models';
+import { mockAppContext } from '../../../models/mocks';
+import { LocationProvider } from '@reach/router';
+
+function renderWith(props?: any) {
+  return render(
+    <AppContext.Provider value={mockAppContext()}>
+      <LocationProvider>
+        <ThirdPartyAuthCallback {...props} />;
+      </LocationProvider>
+    </AppContext.Provider>
+  );
+}
+
+describe('ThirdPartyAuth component', () => {
+  let bundle: FluentBundle;
+  beforeAll(async () => {
+    bundle = await getFtlBundle('settings');
+  });
+  it('renders as expected', async () => {
+    renderWith();
+    expect(
+      screen.getByText(
+        'Please wait, you are being redirected to the authorized application.'
+      )
+    ).toBeInTheDocument();
```

Comment:
FWIW you can remove the `expect` and `toBeInTheDocument`. Not sure you need this to be `async` either.

Do we have an existing functional test for this that we plan to port over?